src/utils.js

```diff
@@ -10,10 +10,8 @@ import queries from './queries'
  * @param {Object} target Target.
  * @returns {Object} Merged object.
  */
-export function mergeConcat(target) {
-  let args = Array.prototype.slice.call(arguments, 1)
-
-  args.unshift(target)
+export function mergeConcat() {
+  let args = Array.prototype.slice.call(arguments, 0)
```

Comment:
Would you be able to use argument spreading here? It's used other places in the code
```
export function mergeConcat(...args) {
```